numpy/__init__.cython-30.pxd

```diff
@@ -329,8 +329,8 @@ cdef extern from "numpy/arrayobject.h":
     ctypedef long double  npy_float128
 
     ctypedef struct npy_cfloat:
-        double real
-        double imag
+        float real
+        float imag
```

Comment:
Note this _is_ the correct definition for `np.cfloat`, but is _not_ the correct definition for `npy_cfloat`